Base/home/anon/tests/run-tests-and-shutdown.sh

```diff
@@ -6,26 +6,46 @@ echo "==== Running Tests on SerenityOS ===="
 run(index) {
     shift
     test_cmd=($*)
-    echo "Running test $index out of $count_of_all_tests -- $test_cmd"
+    echo "==== Running test $index out of $count_of_all_tests -- $test_cmd ===="
+    echo "::group::$test_cmd"
     if $test_cmd {
         echo "::debug file=$test_cmd:: $test_cmd passed!"
     } else {
+        fail_count=$(expr $fail_count + 1)
+        failed_tests=($failed_tests $test_cmd)
```

Comment:
The shell now has the means to let you avoid calling out to `expr` here:
```sh
failed_tests=${concat_lists $failed_test ($test_cmd)}

# later, to get the count
failed_count=${length $failed_tests}
```

Reply:
Vereh nice. Updated to use as suggested.